x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -895,11 +900,6 @@ public enum Cap {
         /**
          * Full text functions can be used in disjunctions
          */
-        FULL_TEXT_FUNCTIONS_DISJUNCTIONS,
```

Comment:
Why was this removed? 🤔 Poor disjunctions...

Reply:
We are not doing disjunctions anymore. It is so 2024

Reply:
Fixed.